humble/distribution.yaml

```
@@ -701,6 +701,25 @@ repositories:
       url: https://github.com/ros2/demos.git
       version: humble
     status: developed
+  deptha-ros:
+    doc:
+      type: git
+      url: https://github.com/luxonis/depthai-ros.git
+      version: ros-release
+    release:
+      packages:
+      - depthai-ros
+      - depthai_bridge
+      - depthai_examples
+      - depthai_ros_msgs
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/luxonis/depthai-ros-release.git
+      version: 2.5.1-1
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/luxonis/depthai-ros.git
```

```suggestion
      url: https://github.com/luxonis/depthai-ros.git
      version: ros-release
    status: developed
```

The merge lost the version and status fields.

---

ah. Sorry my bad. Thanks for suggestion.